scripts/borg.exe.spec

```diff
@@ -51,15 +51,25 @@ exe = EXE(pyz,
           upx=True,
           console=True)
 
-if False:
+if os.environ.get('PYINSTALLER_DIR_BIN', False):
```

Comment:
can we just always build the exe and the dir binary?

iirc the tests check for presence of borg.exe and if it is there, the tests will be invoked using that exe.

so guess we should always have the exe (and maybe just hope that the dir-based binary will also work if the exe works).